src/game_api/layer.cpp

```diff
@@ -93,8 +93,8 @@ Entity* Layer::spawn_entity_over(ENT_TYPE id, Entity* overlay, float x, float y)
 
 Entity* Layer::get_grid_entity_at(float x, float y)
 {
-    const uint32_t ix = static_cast<uint32_t>(x + 0.5f);

```

Comment:
Why this change? I would need to find the offset to convince you perhaps, but this was based on exactly the disasm. I believe the behavior is essentially the same, but maybe not, so curious what your reasoning was.

Reply:
Just to make it clear what it does instead of adding some magic number. Nothing important, i can change it back if you insist 

Also apparently the standard covers more situations, the only difference i found is rounding -0.0 (the +0.5f rounds to 0.0 instead of -0.0), not really important in this situation of course